eventsourcing/src/main/java/org/axonframework/eventsourcing/SnapshotterSpanFactory.java

```diff
@@ -0,0 +1,31 @@
+package org.axonframework.eventsourcing;
+
+import org.axonframework.tracing.Span;
+
+/**
+ * Span factory that creates spans for the {@link Snapshotter}.
+ * You can customize the spans of the snapshotter by creating your own implementation.
+ *
+ * @author Mitchell Herrijgers
+ * @since 4.9.0
+ */
+public interface SnapshotterSpanFactory {
```

Comment:
I initially expected this interface to extend the `SpanFactory`, but seeing the remainder of the PR, I understand why you haven't done that..

Reply:
Yeah that would just be chaining all methods. And, in the new format, components are not expected to interact with those methods at all. Only the `propagateContext`

Reply:
:+1: good call then. I'll resolve this comment.